riscv_private.h

```diff
@@ -138,6 +141,15 @@ struct riscv_t {
     /* user provided data */
     riscv_user_t userdata;
 
+#ifdef ENABLE_GDBSTUB
+    /* gdbstub instance */
+    gdbstub_t gdbstub;
+
+    /* GDB instruction breakpoint */
+    bool bp_is_set;
```

Comment:
The prefix `bp_` might confuse someone who thinks bp = branch prediction. Revise them as following:
* `breakpoint_specified` (bool)
* `breakpoint_addr` (riscv_word_t)